### PR TITLE
Fixed 2227 - Add confirmation for "remove core contributor" 

### PR DESCRIPTION
**Pull request by sudo-kaizen**
This PR solves the issue #2227 - Add confirmation for "remove core contributor" 
using the JavaScript confirm method.
The display message on the confirm dialog is displayed with the intl.formatMessage
as requested in the specification.

Please find attached a picture of the solution below
Screenshot (21)

### CODE REVIEW

File: components/EditMembers.js [hunk 1]
```diff
@@ -51,6 +51,14 @@ class EditMembers extends React.Component {
       },
       'user.email.label': { id: 'user.email.label', defaultMessage: 'email' },
       'user.since.label': { id: 'user.since.label', defaultMessage: 'since' },
+      'members.remove.confirm.messageBegin': {
```
Comment (Betree) [inline components/EditMembers.js]: This should be only one message including the variables in it.

Like `Do you really want to remove {userName} ({email}) from the core contributors of the collective?`

See https://docs.opencollective.com/help/developers/translations#dont-assume-words-order-stays-the-same-in-other-languages

Comment (sudo-kaizen) [reply components/EditMembers.js]: Alright.. Thanks for the feedback.. working on it right now

Comment (sudo-kaizen) [reply components/EditMembers.js]: I just made this fix

Comment (znarf) [reply components/EditMembers.js]: Have you tested that it works as expected? Seems there is a typo.

Comment (sudo-kaizen) [reply components/EditMembers.js]: Oh! 😅 Found it! Fixing it.. You guys are awesome @znarf @Betree

**Comment by sudo-kaizen** [comment]
I've updated the translations files with `npm run build:langs`. Is there any thing I'm missing? @Betree @znarf Thanks

**Comment by Betree** [comment]
@sudo-kaizen It's fine but the script added `\r\n` instead of regular `\n`. I guess you're on Windows :wink: Not your fault and there's not much you can do for that, we'll need to fix it.

**Pull request merged**